cmd/kubelet/app/server.go

```diff
@@ -201,6 +203,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.CloudProvider, "cloud_provider", s.CloudProvider, "The provider for cloud services.  Empty string for no provider.")
 	fs.StringVar(&s.CloudConfigFile, "cloud_config", s.CloudConfigFile, "The path to the cloud provider configuration file.  Empty string for no configuration file.")
 	fs.StringVar(&s.ResourceContainer, "resource_container", s.ResourceContainer, "Absolute name of the resource-only container to create and run the Kubelet in (Default: /kubelet).")
+	fs.StringVar(&s.CgroupParent, "cgroup_parent", s.CgroupParent, "Optional parent cgroup to pass to Docker. All pods will run underneath it.")
```

Comment:
nit: Drop 'docker' since kubelet will support other container runtimes like rocket. I would also mention that this option is enforced on a best effort basis.

@dchen1107: How do we plan to handle runtime specific options? Disjoint flags are probably not a good idea in the log run.